llvm/lib/Support/CommandLine.cpp

```diff
@@ -561,7 +565,19 @@ SubCommand *CommandLineParser::LookupSubCommand(StringRef Name) {
 
     if (StringRef(S->getName()) == StringRef(Name))
       return S;
+
+    unsigned Distance =
+        Name.edit_distance(S->getName(), /*AllowReplacements=*/true,
```

Comment:
In clang and lld, we do something like `if (OptTbl.findNearest(ArgString, Nearest, VisibilityMask) > 1) ` to reject a candidate with a too large edit distance

Reply:
Thanks. Fixed.